torchmultimodal/modules/layers/attention.py

```diff
@@ -0,0 +1,143 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+import itertools
+from typing import Optional
+
+import numpy as np
+
+import torch
+import torch.nn as nn
```

Comment:
Piling on import-related comments, why not `from torch import nn`? I feel that's kinda the standard